src/module/actors/sheets/character-sheet2e.js

```diff
@@ -10,7 +10,14 @@ export class STACharacterSheet2e extends ActorSheet {
       dragDrop: [{
         dragSelector: '.item-list .item',
         dropSelector: null
-      }]
+      }],
+      tabs: [
+        {
+          navSelector: '.sheet-tabs',
```

Comment:
These ids and selectors are a little too generic.  There's probably more tabs in the sheet's future, and the name "sheet body" also implies to me a more significant content area is being changed than just a few fields in the top of a sheet.

Reply:
Selectors renamed to:
character-tabs, character-header, biography and klingon